tests/a_scripts/inference/test_convert_composer_to_hf.py

```diff
@@ -549,14 +740,7 @@ def test_huggingface_conversion_callback(
 
     original_model = COMPOSER_MODEL_REGISTRY[model_cfg['name']](model_cfg,
                                                                 tokenizer)
-
-    optimizer_config = {
-        'name': 'decoupled_adamw',
-        'lr': 6e-4,
-        'betas': [0.9, 0.95],
-        'eps': 1e-8,
-        'weight_decay': 0.0,
-    }
+    optimizer_config = _OPTIMIZER_CFG.copy()
```

Comment:
should be deepcopy, yeah?

Reply:
that's a good point, I'll just make it a lazily generated dict